ui/src/pages/Results.js

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {Component} from 'react';
+import {FormattedMessage, defineMessages, injectIntl} from 'react-intl';
+import {withRouter} from 'react-router-dom';
+import Fab from '@material/react-fab';
+
+import Footer from './../components/Footer.js';
+import LoadingIndicator from './../components/LoadingIndicator.js';
+import RepoHeader from './../components/RepoHeader.js';
+import SearchBar from './../components/SearchBar.js';
+
+const messages = defineMessages({
+  createNewRecord: {
+    id: 'Results.createNewRecord',
+    defaultMessage: 'Create new record',
+    description: 'Label on a button for creating a new database entry.',
+  },
+  noResultsFound: {
+    id: 'Results.noResultsFound',
+    defaultMessage: 'No results found.',
+    description: 'A message shown when a search query has produced no results.',
+  },
+});
+
+/*
+ * A component for showing a search results page.
+ */
+class Results extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoaded: false,
+      error: null,
+      repo: null,
+      results: null
+    };
+    this.goToAdd = this.goToAdd.bind(this);
+    this.loadResults = this.loadResults.bind(this);
+    this.handleSearch = this.handleSearch.bind(this);
+  }
+
+  goToAdd() {
+    this.props.history.push({
+          pathname: '/' + this.props.match.params.repoId + '/create',
+      });
+  }
+
+  loadResults() {
+    const query = new URL(window.location.href).searchParams.get('query_name');
+    var apiCalls = [
+        '/' + this.props.match.params.repoId + '/d/repoinfo',
```

Comment:
Optional: Maybe rename `repoinfo` to `repo` in the URL to be consistent with variable names we use to hold its result?